### PR TITLE
Added other bodies to the bodies module

### LINKED ISSUE #234: Add more Bodies to the bodies module
🐞 **Problem**

Currently, the bodies module only has 2 bodies. The list can be extended.

🎯 **Goal**

A better plotting experience

💡 **Possible solutions**

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by obi-wan-shinobi**
Added all other bodies apart from moon, earth and the sun

Fixes #234 
 @ritzvik

### CODE REVIEW

**Comment by ritzvik** [comment]
For quality checks, follow this
https://docs.einsteinpy.org/en/latest/dev_guide.html#code-linting

**Comment by obi-wan-shinobi** [comment]
> For quality checks, follow this
> https://docs.einsteinpy.org/en/latest/dev_guide.html#code-linting

Okay, I'll keep this in mind before making changes the next tim

**Comment by ritzvik** [comment]
Can you squash the commits into 1 or 2?
Otherwise seems fine

**Comment by obi-wan-shinobi** [comment]
> Can you squash the commits into 1 or 2?
> Otherwise seems fine

File: src/einsteinpy/tests/test_bodies.py [hunk 1]
```diff
@@ -1,66 +1,186 @@
...
+
+
+def test_moon_extras():
+    assert Moon.a == 0
+    assert Moon.name == "Moon"
+    assert Moon.q == 0 * u.C
+
+
+def test_mercury_base_properties():
+    assert Mercury.parent == Sun
```
Comment (ritzvik) [inline src/einsteinpy/tests/test_bodies.py]: Without explicitly writing tests for each of the body we can do it like this
```
pytest.mark.parametrize("obj, parent, R, mass", [(Mercury, Sun, 2439.7 * u.km, 3.285e23 * u.kg), (Venus, Sun, 6051.8 * u.km, 4.867e24 * u.kg)])
def test_predefined_bodies_base_properties(obj, parent, R, mass):
    assert obj.parent == parent
    assert obj.R == R
    assert obj.mass == mass
```

Something similar for `test_..._extras()`.
Read up about pytest `parametrize`

Comment (obi-wan-shinobi) [reply src/einsteinpy/tests/test_bodies.py]: Oh, okay!

**Comment by ritzvik** [comment]
@shreyasbapat Why the git diff is so weird?

**Comment by ritzvik** [comment]
@obi-wan-shinobi Thanks for hanging in there :p

Merging :)

**Pull request merged**